jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.vue

```diff
@@ -35,6 +35,38 @@
       ></v-text-field>
     </v-row>
 
+    <v-div v-if="n_moment > 0 && dataset_spectral_unit !== ''">
+      <v-row>
+        <v-radio-group
+          label="Output Units"
+          hint="Choose whether calculated moment is in units of wavelength or velocity."
+          v-model="output_unit_selected"
+          column>
+          <v-radio
+            v-for="item in output_unit_items"
+            :key="item.label"
+            :label="item.label"
+            :value="item.label"
+          ></v-radio>
+        </v-radio-group>
+    </v-row>
+    <v-row v-if="output_unit_selected === 'Velocity'" align="center">
+      <v-col cols="9">
+        <v-text-field
+        ref="reference_wavelength"
+        type="number"
+        label="Reference Wavelength"
+        v-model.number="reference_wavelength"
+        hint="Rest wavelength of the line of interest"
+        persistent-hint
+        :rules="[() => reference_wavelength !== '' || 'This field is required',
+                 () => reference_wavelength >=0 || 'Wavelength must be positive']"
+        ></v-text-field>
+      </v-col>
+      <v-col cols="3" justify="left">{{dataset_spectral_unit}}</v-col>
+    </v-row>
+    </v-div>
```

Comment:
I have some style suggestions, but they get too messy as a review suggestion, so I'll submit a PR to yours and feel free to decide what you want to keep.  That addresses:
* replaces nonexisting `v-div` with `div`
* removes extra padding around reference wavelength
* moves the unit from a column to a suffix on the text field (I'm torn on this one, we use both patterns in jdaviz though right now, so pick whichever you'd prefer.  But if you'd rather columns, add `class="row-no-outside-padding"` to the `v-row`, use `dataset_spectral_unit.replace('Angstrom', '&#8491;')` and will need to fix the vertical alignment of the unit)
* removes unused vertical space for no hint on the radio buttons
* stacks radio buttons horizontally in a row (to be consistent with links control and to save a little space)
* fixes indentation